ivadomed/keywords.py

```diff
@@ -274,3 +276,58 @@ class SegmentationPairKW:
     INPUT_METADATA = "input_metadata"
     GT = "gt"
     INPUT = "input"
+
+@dataclass
```

Comment:
I'd guess you'd like to add `@dataclass(frozen=True)`

Reply:
Yeah, really need a PR to fix all these actually. Good point.